assets/styles/scss/_variables.scss

```diff
@@ -52,7 +52,7 @@ $CamptonExtraLight: "Campton Extra Light", "Courier New", sans-serif;
 // Colors
 $primary: $nu-purple;
 $link: $nu-purple;
-$info: $blue;
+$info: $rich-black-80;
```

Comment:
Is there a reason this was changed to black ? I tried to ingest a sheet, the colors don't seem to be in sync on the page.
<img width="1398" alt="Screen Shot 2020-04-24 at 4 01 49 PM" src="https://user-images.githubusercontent.com/14085957/80256809-1ae06380-8645-11ea-90d7-9298d830058c.png">

Reply:
Yea, I think the blue just doesn't make sense maybe?  I've noticed in some other Northwestern emails, that they use the NU "rich black" color to highlight information, so was playing around with that idea I suppose.  This will probably change about 5 more times, just experimenting, not a final decision.